gremlinapi/util.py

```diff
@@ -17,10 +17,29 @@ def get_version():
 string_types = (type(b""), type(""), type(f""))
 
 
+def experimental(func):
```

Comment:
Might be nice to take in an optional message that enhance context on why a method is listed as experimental.

Reply:
Agreed. Done.